lms/models/lti_params.py

```diff
@@ -171,34 +171,8 @@ def _to_lti_v11(v13_params):
         v11_params.update({f"custom_{key}": value for key, value in custom.items()})
 
     if "roles" in v11_params:
-        context_roles = []
```

Comment:
This is not needed anymore, this was re-parsing roles to taking "scope" into account.